patches/4D5308BC - Crackdown 2 (TU5).patch.toml

```diff
@@ -0,0 +1,183 @@
+title_name = "Crackdown 2"
+title_id = "4D5308BC" # MS-2236
+hash = "8309EB7166C1B9C3" # default.xex
+#media_id = "18B6CD67" # Disc (World): http://redump.org/disc/14538
+
+[[patch]]
+    name = "Disable Achivements"
+    desc = "Fixes achivement audio loop, note this will prevent the in-game achivement dialog. Improves audio log."
```

Comment:
```suggestion
    name = "Disable Achievements"
    desc = "Fixes achievement audio loop, note this will prevent the in-game achievement dialog. Improves audio log."
```